crates/ruff/src/rules/pyupgrade/rules/extraneous_parentheses.rs

```diff
@@ -134,21 +134,21 @@ fn match_extraneous_parentheses(tokens: &[LexResult], mut i: usize) -> Option<(u
 
 /// UP034
 pub(crate) fn extraneous_parentheses(
+    diagnostics: &mut Vec<Diagnostic>,
```

Comment:
Nit: I would have kept tokens as the first argument because it is what the rules primarily operate on. Diagnostics is more a context argument, similar to settings